client/src/components/ProtectedRoute.tsx

```diff
@@ -0,0 +1,20 @@
+import React, { Component } from 'react';
+import { Route, Redirect } from 'react-router-dom';
+import { useAuth } from '../context/useAuthContext';
+
+const ProtectedRoute = ({ path, component: Component, ...rest }: { path: string; component: any }) => {
+  const loggedInUser = useAuth();
+  return (
+    <Route
```

Comment:
Maybe pass in the `...rest` props from above in addition to the default props that <Route> provides normally? :)

Reply:
resolved.